io.qt.PySide.BaseApp.metainfo.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+    <id>io.qt.PySide6.BaseApp</id>
+    <name>PySide6 BaseApp</name>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-free=https://doc.qt.io/qtforpython-6/</project_license>
```

Comment:
This should be LGPL right? The commercial part is provided as addons afaik and is not being built here?

Reply:
right. The validation with flatpak-builder-lint showed me a warning to use LicenseRef. I have updated it.

Reply:
That's fine, info hints are not errors.